packages/login/src/styles/core.scss

```diff
@@ -0,0 +1,23 @@
+:global {
+  @import 'base';
+  @import 'vendor/normalize';
```

Comment:
pretty much the whole styles folder is boilerplate code from starter-kit. I think we wait until we have a solid solution for styling questions and then update the packages accordingly.